testsuites/listener/shared/client_sg/conftest.py

```diff
@@ -111,6 +111,7 @@ def setup_client_syncgateway_test(request, setup_client_syncgateway_suite):
 
     # Yield values to test case via fixture argument
     yield {
+        "liteserv": liteserv,
```

Comment:
Maybe pass a platform string in LiteServFactory and set .platform property https://github.com/couchbaselabs/mobile-testkit/blob/master/keywords/LiteServBase.py#L18-L22?